ramalama/model.py

```diff
@@ -106,18 +106,18 @@ def serve(self, args):
             exec_args = ["vllm", "serve", "--port", args.port, symlink_path]
 
         if args.generate == "quadlet":
-            return self.quadlet(args, exec_args)
+            return self.quadlet(symlink_path, args, exec_args)
 
         exec_cmd(exec_args)
 
-    def quadlet(self, args, exec_args):
+    def quadlet(self, model, args, exec_args):
         port_string = ""
         if hasattr(args, "port"):
             port_string = f"PublishPort={args.port}"
 
         name_string = ""
-        if hasattr(args, "name") and args.name != "":
-            name_string = f"Name={args.name}"
+        if hasattr(args, "name") and args.name:
+            name_string = f"ContainerName={args.name}"
 
         print(
```

Comment:
Not blocking, but have you considered using `f""` instead of `%`?

Reply:
I have used both in the code,  Eric tends to use the f"" and I tend to use the %s method. But we should go through the code and use one for consistency.

Reply:
I understand the desire for printf-style, that's normally what I lean towards in new languages. But I am starting to prefer f"", you don't have to count out the args. This code is a good example, I really have to count the %'s and the args to figure out which is the 5th parameter. Also with f"" one doesn't have to worry about type.

I'm leaning towards f""

These python style {} formatted strings actually made the C++ standard recently via std::format (which was previously https://github.com/fmtlib/fmt). It's actually faster than printf family of functions.

Reply:
Genuinely I don't mind using both either where appropriate. flake8 and black provides us some standards, not everything has to be standardised on one solution. Like maybe sometimes one does want to explicitly represent something as a %s and that's fine with me!